index.html

```diff
@@ -1034,13 +1038,19 @@ <h2 id="BaseAudioContext">
               <li>In case of failure, queue a task on the <a>control thread</a>
               to execute the following, and abort these steps
                 <ol>
+                  <li>Reject all promises from <a>pendingResumePromises</a> in
+                  order, then clear <a>pendingResumePromises</a>.
+                  </li>
                   <li>Reject <em>promise</em>
```

Comment:
Perhaps I'm confused, but lines 1017-1018 says that if the context is not allowed, the promise is appended to pendingResumePromises. And if the control message is not queued, how is pendingResumePromises ever handled?